Tests/CodegenCLITests/Commands/InitializeTests.swift

```diff
@@ -71,11 +94,69 @@ class InitializeTests: XCTestCase {
     expect(command.schemaName).to(equal("ShortFormatSchemaName"))
   }
 
+  func test__parsing__givenParameters_moduleNameLongFormat_shouldParse() throws {
```

Comment:
While these tests are awesome and super comprehensive, remind me to have a conversation with you about my views on testing behavior vs configuration! Would love to save you some time in the future possibly.